sci-electronics/drahnr-oregano/files/drahnr-oregano-0.84.39-fix-gschema.patch

```diff
@@ -0,0 +1,11 @@
+--- drahnr-oregano-76560fa/src/wscript.orig	2018-08-10 23:37:27.539066687 +0300
++++ drahnr-oregano-76560fa/src/wscript	2018-08-10 23:38:07.330065511 +0300
+@@ -41,7 +41,7 @@
+ 		includes = ['.', 'tools/', 'engines/', 'gplot/', 'model/', 'sheet/'],
+ 		use = 'shared_objects',
+ 		uselib = 'M XML GOBJECT GLIB GTK3 XML GOOCANVAS GTKSOURCEVIEW3',
+-		settings_schema_files = ['../data/settings/'+bld.env.gschema_name ] if not bld.options.no_install_gschema else [],
++#		settings_schema_files = ['../data/settings/'+bld.env.gschema_name ] if not bld.options.no_install_gschema else [],
```

Comment:
Can't you just enable the option that disables it instead of patching it out?

Also for patches, don't comment stuff out because that makes no sense. Just remove it.